PIL/ImageOps.py

```diff
@@ -178,6 +178,27 @@ def crop(image, border=0):
         )
 
 
+def scale(self, factor, resample=Image.NEAREST):
+    """
+    Returns a rescaled image by a specific factor given in parameter.
+    A factor greater than 1 expands the image, between 0 and 1 contracts the
+    image.
+
+    :param factor: The expansion factor, as a float.
+    :param resample: An optional resampling filter. Same values possible as
+       in the PIL.Image.resize function.
+    :returns: An :py:class:`~PIL.Image.Image` object.
+    """
+    if factor == 1:
+        return self._new(self.im)
```

Comment:
I applaud the thought, but `resize` has it's own check for the image being the same size - https://github.com/python-pillow/Pillow/blob/master/PIL/Image.py#L1539 - so I'm going to suggest that this is redundant? Feel free to disagree.